web/yo/app/scripts/controllers/gene.js

```diff
@@ -1457,6 +1494,22 @@ angular.module('oncokbApp')
                 return list.join(', ');
             };
 
+            $scope.getOldCancerTypesName = function(cancerTypes) {
```

Comment:
Yeah right. But I can not use that function since it is getting data from a collaborative list and collaborative string. Here I am getting data from Javascript array and string variables